### ISSUE TITLE
Ui5-tabcontainer a11y issue.

### ISSUE THREAD

**Issue by lateefsofi** [SAP/ui5-webcomponents#1977]
**Describe the bug**
We use Axe for testing our components. We have implemented ui4-tabcontainer in ReactJs and we get accessibility issues for a button "Buttons must have discernible text". 

**To reproduce**
1. Go to http://mshaik.ies.c.na-us-1.cloud.sap:3001/hp-blended-item/COURSE/1591207334000
2. You should have axe chrome tool installed
3. Turn the screen into responsive mode(Mobile).
3. Open dev tools, then go to Axe and Click Analyze.
4. You should be able to see the issue "Buttons must have discernible text"

**Expected behavior**
Add aria-label or aria-labelledby attribute so that there is no a11y violation.

**Screenshots**
![image](https://user-images.githubusercontent.com/18447407/88258434-c8fbb800-ccdd-11ea-9a0f-ed810124cee0.png)

**Context**
- UI5 Web Components version: "@ui5/webcomponents": "0.21.7",
- OS/Platform: Windows
